doc/nrf/working_with_nrf/nrf70/developing/regulatory_support.rst

```diff
@@ -0,0 +1,358 @@
+.. _ug_nrf70_developing_regulatory_support:
+
+Operating with regulatory support
+#################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The nRF70 Series devices operate in free radio spectrum bands (2.4 GHz and 5 GHz).
+The supported channels in each band need to adhere to worldwide regulatory operation rules.
+The regulatory rules vary based on the country.
+The rules are framed on the basis of the following important parameters:
+
+* Channel scan mode (Active/Passive)
+* Dynamic Frequency Selection (DFS) support
+* Maximum transmit power
+
+Regulatory Database
+*******************
+
+The regulatory database used by the nRF70 Series captures regulatory rules in 173 different countries.
+For more information about the individual country rules, see Wi-Fi regulatory channel rules.
+In addition, the regulatory database has an entry for a world regulatory domain, which has highly restrictive rules making it suitable for the worldwide operation.
+
+The regulatory database is a part of the nRF70 Series ROM based firmware images.
+You can configure the country of operation and based on this configuration, the appropriate rules of operation are enforced.
+
+Supported 20 Mhz channels
```

Comment:
I believe these are supposed to be part of the Product specification. Please confirm with Rajeshwar.

Reply:
@rajb9: Please confirm.

Reply:
For the time being let it be in the NCS doc. Will decide it later.